crates/turborepo-lib/src/config.rs

```diff
@@ -132,6 +132,14 @@ pub enum Error {
         #[source_code]
         text: NamedSource,
     },
+    #[error("found `pipeline` field instead of `tasks`")]
+    #[diagnostic(help("In turbo 2.0, `pipeline` has been renamed to `tasks`"))]
```

Comment:
```suggestion
    #[diagnostic(help("Deprecated in 2.0: `pipeline` has been renamed to `tasks`"))]
```

Reply:
technically not deprecated since it doesn't work at all, no?